modules/custom_operations/user_ie_extensions/tokenizer/python/pyproject.toml

```diff
@@ -0,0 +1,45 @@
+[project]
+name = "ov_tokenizer"
+version = "0.0.1"
+description = "Convert tokenizers into OpenVINO models"
+requires-python = ">=3.8"
+authors = [
+    { name = "Artur Paniukov", email = "artur.paniukov@intel.com" },
+    { name = "Sergey Lyalin", email = "sergey.lyalin@intel.com" },
```

Comment:
I suppose concrete authors are not required.
We can use generic `OpenVINO Developers <openvino@intel.com>`